app/data_loading.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+import json
```

Comment:
There is no need to place blank line between imports that belong to a similar group